### PR TITLE
[ENH] Gamma Distribution

### PR DESCRIPTION
**Pull request by ShreeshaM07**
#### What does this implement/fix? Explain your changes.

Implements Gamma distribution.
#### Does your contribution introduce a new dependency? If yes, which one?
No

#### Did you add any tests for the change?
Yes

##### For all contributions
- [ ] I've added myself to the [list of contributors](https://github.com/sktime/skpro/blob/main/CONTRIBUTORS.md) with any new badges I've earned :-)
  How to: add yourself to the [all-contributors file](https://github.com/sktime/skpro/blob/main/.all-contributorsrc) in the `skpro` root directory (not the `CONTRIBUTORS.md`). Common badges: `code` - fixing a bug, or adding code logic. `doc` - writing or improving documentation or docstrings. `bug` - reporting or diagnosing a bug (get this plus `code` if you also fixed the bug in the PR).`maintenance` - CI, test framework, release.
  See here for [full badge reference](https://allcontributors.org/docs/en/emoji-key)
- [x] The PR title starts with either [ENH], [MNT], [DOC], or [BUG]. [BUG] - bugfix, [MNT] - CI, test framework, [ENH] - adding or improving code, [DOC] - writing or improving documentation or docstrings.

##### For new estimators
- [x] I've added the estimator to the API reference - in `docs/source/api_reference/taskname.rst`, follow the pattern.
- [x] I've added one or more illustrative usage examples to the docstring, in a pydocstyle compliant `Examples` section.
- [x] If the estimator relies on a soft dependency, I've set the `python_dependencies` tag and ensured
  dependency isolation, see the [estimator dependencies guide](https://www.sktime.net/en/latest/developer_guide/dependencies.html#adding-a-soft-dependency).

### CODE REVIEW

File: skpro/distributions/gamma.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+class Gamma(_ScipyAdapter):
+    r"""Gamma Distribution.
+
+    The Gamma Distribution is parameterized by shape :math:`\alpha` and
+    rate :math:`\beta`, such that the pdf is
+
+    ..math:: f(x) = \frac{x^(\alpha-1)*\exp\left(-\beta*x)*\beta^\alpha}{\tau(\alpha)}
+
+    where `\tau(\alpha)` is the Gamma function.
+    For all positive integers,`\tau(\alpha) = (alpha-1)!`.
```
Comment (VascoSch92) [inline skpro/distributions/gamma.py:21]:
```suggestion
    where :math:`\tau(\alpha)` is the Gamma function.
    For all positive integers, :math:`\tau(\alpha) = (\alpha-1)!`.
```

**Review by fkiraly — APPROVED** [review]
Looks good to me.

Typos in docstring should be fixed, though are not blocking.

File: skpro/distributions/gamma.py (hunk 1 above)
Comment (ShreeshaM07) [reply skpro/distributions/gamma.py]: Could you please be so kind to write the expression for the `f(x)` as well in this LaTex format I am not able to do it with trial and error, or maybe lead me to some resource that explains the usage.

Comment (fkiraly) [reply skpro/distributions/gamma.py]: but `f` looks correct already - does it not compile to sth readable?

You can always test it by putting it in markdown, like this:

$`f(x) = \frac{x^{\alpha-1}\exp\left(-\beta x\right)\beta^{\alpha}}{\tau(\alpha)}`$

Comment (ShreeshaM07) [reply skpro/distributions/gamma.py]: for some reason the expression is working when i use $$ in markdown here but it doesnt convert in readthedocs and instead if i use ..math:: neither here nor there it is readable.

**Comment by ShreeshaM07** [comment]
Finally it works! please let me know if anything else needs to be done.

File: skpro/distributions/gamma.py (hunk 1 above)
Comment (fkiraly) [reply skpro/distributions/gamma.py]: I'm looking at the docs, it is perfectly readable for me:

image

**Comment by fkiraly** [comment]
oh, I see - you fixed it. Great!

**Pull request merged**